packages/big-design/src/components/Tree/types.ts

```diff
@@ -0,0 +1,42 @@
+export interface TreeProps<T> {
+  iconless?: boolean;
+  nodes: TreeNodeProps<T>[];
+  selectable?: 'radio' | 'multi';
+  onExpand?(node: TreeNodeProps<T>): Promise<TreeNodeRef<T>> | TreeNodeRef<T> | void;
+  onCollapse?(node: TreeNodeProps<T>): Promise<TreeNodeRef<T>> | TreeNodeRef<T> | void;
+  // Based on the selectable prop, it can either
+  // return and array of values, or just one.
+  onSelect?(values: T[] | T): void;
```

Comment:
Would it be useful to also return the IDs that are selected? This is because `value` alone cannot sufficiently indicate which items are selected.

Reply:
In most cases we would map the `id` with the `value`. Could you provide a use-case in which value is not sufficient?

Reply:
I raised this because `value` is not guaranteed to be unique. As far as I can tell from the doc, only `id` is unique. If it's not unique, then you can't find the corresponding `id` with `value`. And if it's unique, then you don't really need an additional `id` as an identifier.